api/src/db/database.rs

```diff
@@ -3173,6 +3173,30 @@ impl Database {
     .fetch_all(&self.pool)
     .await
   }
+
+  pub async fn get_recent_packages_by_user(
+    &self,
+    user_id: &uuid::Uuid,
+  ) -> Result<Vec<Package>> {
+    // need to fetch all scopes where user is in
+    // then fetch all packages in those scopes
+    // then sort by created_at
+    // then limit 10
+    let scopes = self.get_member_scopes_by_user(user_id).await?;
+
+    let mut packages = Vec::new();
+    for scope in scopes {
+      let (_, scope_packages) = self
+        .list_packages_by_scope(&scope.scope, true, 0, 100)
+        .await?;
+      packages.extend(scope_packages.into_iter().map(|(p, _, _)| p));
+    }
+
+    packages.sort_by(|a, b| b.created_at.cmp(&a.created_at));
+    packages.truncate(10);
+
+    Ok(packages)
+  }
```

Comment:
please create an actual sql query instead of this

Reply:
like on big query ?